.npmrc

```diff
@@ -1 +1,2 @@
-registry = https://registry.npmjs.org
+//npm.pkg.github.com/:_authToken=${GH_PACKAGES_NPM_TOKEN}
+@compassion-technology:registry=https://npm.pkg.github.com
```

Comment:
Is this publishing a public package on our private GitHub NPM registry, or is this required to get to publish as @compassion-technology on the main NPM registry? I don't get it yet.

Reply:
From what I read its required to publish to `https://github.com/compassion-technology`. that or we'd have to run `npm login` or something which we don't want. 

It should still be public because I passed that flag in on the `npm publish` command. I had an article pulled up at one point supporting that but I've since closed it now